apps/evm/test/block/header_test.exs

```diff
@@ -1,7 +1,119 @@
 defmodule Block.HeaderTest do
   use ExUnit.Case, async: true
   doctest Block.Header
+
+  import ExthCrypto.Math, only: [hex_to_bin: 1, hex_to_int: 1]
+
   alias Block.Header
+  alias EVM.EthereumCommonTestsHelper, as: Helper
+
+  @forks_with_block %{
+    frontier: 1,
+    homestead: 1_150_000,
+    dao_fork: 1_920_000,
+    eip_150: 2_463_000,
+    spurious_dragon: 2_675_000,
+    byzantium: 4_370_000
+  }
+
+  @next_fork_block_number @forks_with_block[:homestead]
```

Comment:
are we going to run `Homestead` difficulty tests in next pr?

Reply:
Yeah, I figured I'd break the work apart by adding the tests and passing Frontier, and then doing the Homestead work as part of https://github.com/poanetwork/mana/issues/343